packages/mdc-snackbar/mdc-snackbar.scss

```diff
@@ -129,20 +129,26 @@
   /* stylelint-enable plugin/selector-bem-pattern */
 
   &__action-button {
+    @include mdc-typography(button);
     @include mdc-theme-prop(color, secondary);
 
     @include mdc-theme-dark(".mdc-snackbar") {
       @include mdc-theme-prop(color, primary);
     }
 
-    @include mdc-rtl-reflexive-box(margin, right, -16px, ".mdc-snackbar");
```

Comment:
Why was this line removed? This effectively shifts the action button to the left 16px.

Reply:
This line used to be the mechanism to reverse `padding` introduce by button, since now there is no longer `padding: 0 16px`, we also could safely remove the `margin-direction: -16px` here. 

Could you take a look at the deployed dev server and see if this style looks correct to you?